2017-01-13 18:29:14,240 - knowledge_logger - main.py-handle-api- : list index out of range 

2017-01-13 18:29:14,271 - knowledge_logger - main.py : Traceback (most recent call last):
  File "/home/thickpadi5/Project-msft/compfie/Know-src-server/server/main.py", line 211, in handle_api_request
    response_data = unbound_method(self, request_data, _db)
  File "/home/thickpadi5/Project-msft/compfie/Know-src-server/server/main.py", line 358, in handle_techno_transaction
    return controller.process_techno_transaction_request(request, db)
  File "/home/thickpadi5/Project-msft/compfie/Know-src-server/server/controller/technotransactioncontroller.py", line 99, in process_techno_transaction_request
    result = process_get_groupadmingroup_unit_list(db, user_id)
  File "/home/thickpadi5/Project-msft/compfie/Know-src-server/server/controller/technotransactioncontroller.py", line 190, in process_get_groupadmingroup_unit_list
    groupadmin_groupsList = get_groupadmin_registration_grouplist(db, session_user)
  File "/home/thickpadi5/Project-msft/compfie/Know-src-server/server/database/technotransaction.py", line 456, in get_groupadmin_registration_grouplist
    return return_groupadmin_registration_grouplist(groupadmin_grouplist)
  File "/home/thickpadi5/Project-msft/compfie/Know-src-server/server/database/technotransaction.py", line 460, in return_groupadmin_registration_grouplist
    for groups in groupslist[1]:
IndexError: list index out of range
 

